src/Page/Shared/View.elm

```diff
@@ -6,22 +6,29 @@ import Html.Styled.Attributes exposing (attribute, autoplay, css, href, src, tit
 import I18n.Keys exposing (Key(..))
 import I18n.Translate exposing (Language, translate)
 import Message exposing (Msg)
-import Page.Shared.Data
+import Page.Shared.Data exposing (VideoMeta)
```

Comment:
Note that you don't need to expose if you ref it with the namespace when used. Exposing means you could use as `VideoMeta` rather than `Page.Shared.Data.VideoMeta` but with most thing we are going with fully qualified so that source is more explicit in the places imports are used.